src/api/versions/v1/services/moderation-service.ts

```
@@ -0,0 +1,41 @@
+    await this.kvService.setUser(userId, user);
+  }
+
+  public async unbanUser(userId: string): Promise<void> {
+    const user = await this.kvService.getUser(userId);
+
+    if (user === null) {
+      throw new ServerError("USER_NOT_FOUND", "User not found", 404);
+    }
+
+    if (user.ban) {
+      delete user.ban;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)